README.md

```diff
@@ -1,173 +1,361 @@
+- Helm
+    - https://helm.sh/docs/intro/install/
+- A valid authentication token for `registry1.dso.mil`
+- Clone https://github.com/309theddge/opal-helm
 
-## Deploying ArgoCD 
+#### Additional Prerequisites for Local Deployment
 
-The following packages are required before proceeding 
- - minikube 
+- Minikube
     - https://minikube.sigs.k8s.io/docs/start/
-- docker 
+- Docker (DO NOT install Docker Desktop if using Linux)
     - https://docs.docker.com/engine/install/
-- kubectl 
-    - https://kubernetes.io/docs/tasks/tools/
-- helm 
-    - https://helm.sh/docs/intro/install/
+- Clone https://github.com/309theddge/k8s-utils
+- Clone https://github.com/309theddge/argo
+
+#### Optional Extras
+
+- Prometheus Stack
+    - https://github.com/309theddge/kube-prometheus
 
-- You may need more packages depending on your current environment setup
+### Local Installation
 
-### Starting minikube
-From a terminal with administrator access run the following command:
->$minikube start
+#### Starting Minikube
+
+Run the following command in your terminal:
+
+``` bash
+minikube start
+```
 
 If minikube fails to start, use the minikube documentation to troubleshoot any issues 
 https://minikube.sigs.k8s.io/docs/start/
 
-### Setting up ArgoCD regcred
- change directory to the regcred-init directory and run the following command:
+#### ArgoCD
 
->$helm install argoregcred . --create-namespace -n argocd
+ArgoCD is a continuous deployment (CD) tool used to manage and provide automatic updates for Kubernetes applications
 
-### Install ArgoCD helm chart
-now that the argoCD values are configured properly for a local deployment, helm install the argocd from the parent directory of ArgoCD repository
+The opal team uses Argo CD to ensure OPAL deployments are always up-to-date
 
->$helm install argo ./chart -n argocd
 
-### Access the ArgoCD secret
-To access the password for the argocd login you will need to run the following command:
-> $kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+##### Setting up ArgoCD regcred
 
-the output from this command will be used on the argoCD dashboard interface
+`cd` into `k8s-utils/regcred-init` and run the following command:
 
-### Add Repo to ArgoCD Dashboard
-In order to get argoCD dashboard working before applications are added, you need to start a port-forward service with kubectl:
->$kubectl port-forward service/argo-argocd-server -n argocd 8080:443
+``` bash
+helm install argo-regcred . --create-namespace -n argocd -f argo-values.yaml
+```
 
-Access the argoCD interface by opening a browser and navigate to the following:
-> localhost:8080
+##### Install ArgoCD helm chart
 
-there should be a login page for argoCD
->username: **admin**
->password: output from the last section (remove the trailing '%' symbol if present)
+Now that the registry credentials have been configured, `cd` into the `argo` repository cloned previously, and install the helm chart:
```

Comment:
Specify that when running the 'helm install argo . -n argocd' command, you should be in the 'argo/chart' directory.